alphagov/paas-cf
# [#146920941] Add new building IPs to admin IP range

## What

So that we can continue to access the platform when in the new building.

## How to review

This is probably best reviewed along with the corresponding [paas-bootstrap PR](https://github.com/alphagov/paas-bootstrap/pull/69).

* Verify the list of IPs match [the documented list](https://sites.google.com/a/digital.cabinet-office.gov.uk/gds-internal-it/news/aviationhouse-sourceipaddresses)
* Run the pipeline from this branch to verify all works.

## Who can review

Not me.
